pkg/cmd/root.go

```diff
@@ -132,11 +133,30 @@ func init() {
 			"An existing file will be overwritten. "+
 			"Specify \"-\" to log to stderr. "+
 			"Defaults to not logging.")
+
 	RootCmd.SetHelpCommand(&cobra.Command{Hidden: true})
 
+	Walk(RootCmd, func(c *cobra.Command) {
+		alias := c.NameAndAliases()
```

Comment:
please change c to command as a variable name

Reply:
+1